packages/radix-ui-themes/src/components/base-tab-list.css

```diff
@@ -0,0 +1,140 @@
+.rt-BaseTabList {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.rt-BaseTabListLink {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  flex-shrink: 0;
+  position: relative;
+  user-select: none;
+}
+
+.rt-BaseTabListLinkInner,
+.rt-BaseTabListLinkInnerHidden {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rt-BaseTabListLinkInner {
+  position: absolute;
+
+  :where(.rt-BaseTabListLink[data-state='inactive'], :not(.rt-TabNavLink[data-active])) & {
```

Comment:
- Kinda bummer that we need to deal both with `data-state="active"` and `data-active`
- I believe that instead of `:not(.rt-TabNavLink[data-active])` we want `.rt-TabNavLink:not([data-active])`, otherwise you are selecting every element that is not an active tab link (vs every tab link that is not active)